### PR TITLE
add authrealm description

### PR DESCRIPTION
**Pull request by robinbobbitt**

### CODE REVIEW

File: quick_start.adoc [hunk 1]
```diff
@@ -26,6 +26,42 @@ We will first walk through setting up your managed cluster set and placement, th
...
+  namespace: identity-config-authrealm
+spec:
+  identityProviders:
+  - github:
+      clientID: 41c30d83fd1b4774f871
+      clientSecret:
+        name: identitatem-github-client-secret
+      organizations:
+      - identitatem
+    mappingMethod: claim
```
Comment (chrisahl) [inline quick_start.adoc:50]: I would change this from claim to add

File: quick_start.adoc [hunk 2]
```diff
@@ -26,6 +26,42 @@ We will first walk through setting up your managed cluster set and placement, th
...
+metadata:
+  name: identity-config-developers
+  namespace: identity-config-authrealm
+spec:
+  identityProviders:
+  - github:
+      clientID: 41c30d83fd1b4774f871
+      clientSecret:
+        name: identitatem-github-client-secret
+      organizations:
```
Comment (chrisahl) [inline quick_start.adoc:48]: We might want to not have the organizations section in the example as we did see some issue with it in and less problems with it not specified.

File: quick_start.adoc (hunk 1 above)
Comment (robinbobbitt) [reply quick_start.adoc:50]: `claim` is the default value and is more restrictive than `add`. I feel like if someone is going to choose `add` they should do so thoughtfully. @tpouyer thoughts?

File: quick_start.adoc (hunk 2 above)
Comment (robinbobbitt) [reply quick_start.adoc:48]: @vidyanambiar has been researching and is going to add explanation on how to make it work.

File: quick_start.adoc (hunk 1 above)
Comment (tpouyer) [reply quick_start.adoc:50]: I'm going to agree with @robinbobbitt here.  `claim` is the default in OCP and `add` brings with it certain "security concerns" that we don't want to "default" a customer to accepting.

Comment (chrisahl) [reply quick_start.adoc:50]: Ok, works for me.  But you may want to add a troubleshooting topic for claim/add and user

File: quick_start.adoc (hunk 2 above)
Comment (chrisahl) [reply quick_start.adoc:48]: ok

**Review by chrisahl — APPROVED** [review]

**Pull request merged**